ambient_toolbox/services/custom_scrubber.py

```diff
@@ -44,7 +48,7 @@ def process(self):
         self._logger.info('Validating setup...')
         if not self._validation():
             self._logger.warning('Aborting process!')
-            return False
+            raise ScrubbingError('Scrubber Settings validation failed')
```

Comment:
```suggestion
            raise ScrubbingError("Scrubber settings validation failed")
```

Reply:
Warum willst du hier double-quotes machen? Der Rest deiner Datei hat immer single-quotes für Strings (siehe eine Zeile weiter oben in der logger-Warnung)

Reply:
dann halt single 😆